### PR TITLE
[PIN-3445]Integrated support role to registry process 

### PR DESCRIPTION
**Pull request by rGregnanin**

### CODE REVIEW

**Review by galales — APPROVED** [review]
same considerations for PR renaming 🙏

**Comment by beetlecrunch** [comment]
LGTM, I noticed that the authorization test for the `getAttributes` function is completely missing.
Could you add it (src/test/resources/authz.json)?
Thank you!

File: src/test/resources/authz.json [hunk 1]
```diff
@@ -25,16 +26,29 @@
...
+      "verb": "GET",
+      "roles": [
+        "admin",
+        "api",
+        "security",
+        "m2m",
+        "support"
+      ],
+      "route": "getAttributes"
+    },
```
Comment (TonioGela) [inline src/test/resources/authz.json:44]: Of course this change is not enough, you should add the test that uses this json value in `AttributeApiServiceAuthzSpec.scala`

**Review by TonioGela — CHANGES_REQUESTED** [review]

File: src/test/resources/authz.json (hunk 1 above)
Comment (rGregnanin) [reply src/test/resources/authz.json:44]: Sorry for the misunderstanding, next time I pay more attention

**Review by TonioGela — APPROVED** [review]

**Review by beetlecrunch — APPROVED** [review]

**Pull request merged**